css/styles.css

```diff
@@ -81,6 +86,7 @@
 }
 
 .ab-column h6 {
+    clear: both;
```

Comment:
Clearing the headlines of a (floating) column-set could be considered a best-practice (well floating columns are not bleeding edge anymore, but...). So restoring this line of code seemed like a good idea to me, but I'm open to counter arguments.